phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         syntaxCheck="true">
+    <testsuites>
+        <testsuite name="unit">
+            <directory suffix="Test.php">./Tests</directory>
```

Comment:
> it is a common practice to have symfony bundles not contain a src

Well, I don't really like that practice. Mainly because the root always contains lots of other project-related files which this way mixes with source files. Also, this way Test files are always added to the autoloader.

However this is going to be a symfony bundle, so I guess it is more convenient for people to use a bundle which follows common symfony patterns. I leave this up to you.

> the name is the unit tests suite. you mean something like "HttpClientBundle Unit Tests" ?

Yeah, it is again personal preference, but I like longer, but more descriptive names.